.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
check with luke that it's ok to commit ide configurations to the git repo

Reply:
This was missed by my .gitignore changing it.

Reply:
This file is still here